app/konnect/api/identity-management/identity-integration.md

```diff
@@ -0,0 +1,170 @@
+
+   The response body will contain a list of available roles: 
+
+    ```
+    {
+	"analytics": {
+		"name": "Analytics",
+		"roles": {
+			"analytics_admin": {
+				"name": "Analytics Admin",
+				"description": "Allows users to view, create, modify, and delete all content in Konnect Analytics."
+			},
+			"analytics_viewer": {
+				"name": "Analytics Viewer",
+				"description": "Allows users to view all but not modify any content in Konnect Analytics."
+			}
+		}
+	}
+    ```
+
+2. Assign a role to a team by issuing  a `POST` request:
+    
+    The request should contain the `id` as a parameter in the URL. This request requires a JSON response body that contains `role`, `entity_id`, `entity_type`, and `entity_region`. 
+
+        curl --request POST \
+        --url https://global.api.konghq.tech/v2/teams/TEAM_ID/assigned-roles \
+        --header 'Content-Type: application/json' \
+        --data '{
+        "role": "analytics_viewer",
+        "entity_id": "e67490ce-44dc-4cbd-b65e-b52c746fc26a",
+        "entity_type": "runtime_groups",
+        "entity_region": "eu"
+        }'
+    If the information in the request was correct, the response body will return a `200` and the `id` value for the new role. 
+
+    {:.note}
+    > `entity_id` can be found in the {{site.konnect_short_name}} in the **Runtime Instances** section. 
+
+## Assign a user to a custom team
+
+For a user to access the roles assigned to a custom team, the user must become a member of the team. A user may be a part of multiple teams and will inherit all of the roles from the teams they belong to.
+
+1. Obtain a list of users by issuing a `GET` request:
+
+        curl --request GET \
+        --url https://global.api.konghq.tech/v2/users
+    This will provide you with 
+    {
+	"meta": {
+		"page": {
+			"number": 1,
+			"size": 10,
+			"total": 22
+		}
+	},
+	"data": [
+		{
+			"id": "69c60945-d42a-4757-a0b2-c18500493949",
+			"email": "user.email@konghq.com",
+			"full_name": "user",
+			"preferred_name": "User2",
+			"active": true,
+			"created_at": "2022-10-12T16:22:53Z",
+			"updated_at": "2022-10-19T15:18:11Z"
+		},
+
+2. Using the `id` field from the desired user and the `id` field from the team construct and issue a `POST` request: 
+
+        curl --request POST \
+        --url https://global.api.konghq.tech/v2/teams/TEAM_ID/users \
+        --header 'Content-Type: application/json' \
+        --data '{
+        "id": "USER_ID"
+        }'
+
+You will receive a `200` with no response body confirming that the user was added to the custom team. 
+
+
+## Mapping IdP groups to teams
+
+If [single sign on](/konnect/org-management/okta-idp/) is enabled, an organization can optionally enable groups to team mappings. This mapping allows {{site.konnect_short_name}} to automatically map a user to a team according to the group claims provided by the IdP upon login.
+
+1. To update a team mapping, first view all of the available groups:  
```

Comment:
These aren't "available groups": These are groups the user has already mapped to Konnect teams. We don't know the available groups because we don't have access to their IdP.